zingolib/src/lightclient/send.rs

```diff
@@ -53,7 +60,26 @@ impl LightClient {
             .collect()
     }
 
-    //TODO: Add migrate_sapling_to_orchard argument
+    /// Unstable function to expose the zip317 interface for development
+    // TODO: add correct functionality and doc comments / tests
+    #[cfg(feature = "zip317")]
+    pub async fn do_propose(
+        &self,
+        _address_amount_memo_tuples: Vec<(&str, u64, Option<MemoBytes>)>,
+    ) -> Result<Proposal<FeeRule, NoteRecordIdentifier>, String> {
+        use crate::test_framework::mocks::ProposalBuilder;
+
+        Ok(ProposalBuilder::default().build())
+    }
+
+    /// Unstable function to expose the zip317 interface for development
+    // TODO: add correct functionality and doc comments / tests
+    #[cfg(feature = "zip317")]
+    pub async fn do_send_proposal(&self) -> Result<Vec<TxId>, String> {
+        Ok(vec![TxId::from_bytes([0u8; 32])])
```

Comment:
i created a helper function in https://github.com/zingolabs/zingolib/pull/942/files called default_txid